### ISSUE TITLE
Display quick help with hotkeys

### ISSUE THREAD

**Issue by skjerns** [ahrm/sioyek#145]
As a new user it takes a while to memorize all the amazing features and shortcuts of sioyek. Going back to the tutorial PDF every time is not very efficient.

It would be very useful to have a overlay with all hotkeys very briefly listed, e.g. semi-transparent and toggle-able by pressing F1. What do you think?

**Comment by ahrm** [comment]
Maybe we could do that in the future. As an alternative for now, if you press `:` we show a list of all available commands with the corresponding shortcut on the right side. Also we recently added fuzzy search to this list in bb571d01878b33bf53bf72b55b0b9dce394627c9 which should help.

**Comment by anarcat** [comment]
i did this in `~/.config/sioyek/prefs_user.config`:

```
new_command open_tutorial sioyek /usr/share/sioyek/tutorial.pdf
```

and in `~/.config/sioyek/keys_user.config`:

```
open_tutorial <f1>
```

**Comment by ahrm** [comment]
Note that custom command names must start with an underscore.

**Comment by anarcat** [comment]
> Note that custom command names must start with an underscore.

why?

**Comment by ahrm** [comment]
In order not to be confused with future built in sioyek commands. For example in the future I may add a built-in `open_tutorial` command and it could break user configs.

**Comment by anarcat** [comment]
Right, I guessed as much, but in this specific case I very much hope you
will add a open_tutorial command and I *do* hope my command will then
break, so I can remove the cruft from my config. :)
